Sources/ScipioKit/Producer/PIF/XCBuildClient.swift

```diff
@@ -108,3 +113,36 @@ extension DescriptionPackage {
             .appending(components: self.name, target.name)
     }
 }
+
+private struct XCBuildOutputDecoder: ErrorDecoder {
```

Comment:
`xcbuild` output numbers(progress?) and JSON string.

Scipio ignores numbers output and re-format JSON strings.